index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>title</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <div class="toolbar">
+        <span>John Doe</span>
+        <img src="./assets//menu.png" />
+      </div>
+
+      <section class="headline">
+        <span class="title">Hey there. <br />I'm Alex</span>
+        <span class="sub-title">I'am a software Developer</span>
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+          <img src="./assets/github.png" alt="github" />
+          <img src="./assets/linkedin.png" alt="linkedin" />
+          <img src="./assets//Vector.png" alt="vector1" />
+          <img src="./assets//twitter.png" alt="twitter" />
+          <img src="./assets//Vector2.png " alt="vector2" />
```

Comment:
- [ ] The list of icons should be a `ul` tag to group the images, and have a `li` tag per icon; each icon has a link (`a` tag). 
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using `href`.